code/string_algorithms/src/reverse_word_string/reverse_word_string.cpp

```diff
@@ -0,0 +1,48 @@
+// Reverse the words in a given string
+#include <algorithm>
+#include <cstdlib>
+#include <cstring>
+#include <iostream>
+#include <vector>
+
+void reverseWords(std::string s) {
+    std::string temp;
+    int c = 1;
+    s.resize(s.length());
+    temp.resize(s.length());
+    for (int i = 0; i < s.length(); ++i) {
+        if (isspace(s[i]))
+            c++;
+    }
+    std::vector<std::string> str(c);
+    int k = 0, ind = 0;
+    for (int i = 0; i < s.length(); ++i) {
+        temp[k] = (s[i]);
+        k++;
+        if ((isspace(s[i])) || i == s.length() - 1) {
+            str[ind] = temp;
+            temp.clear();
+            temp.resize(s.length());
+            ind++;
+            k = 0;
+        }
+    }
+    std::cout << "After Reversal of Words in String : \n";
+    std::reverse(str.begin(), str.end());
+    for (std::string s : str)
```

Comment:
Just return str and don't print anything in this function.

Reply:
done